### ISSUE TITLE
Remove all _mocks

### ISSUE THREAD

**Issue by pwalsh** [openspending/os-datastore-cli#21]
# Description

The POC of this tool was written before anything else for OpenSpending Next existed... including a working title :). Therefore, there are some classes that are intended to mock core services. They are all located in `_mock`, and are no longer needed in the code.

# Tasks

*Requires: https://github.com/openspending/os-authz-service/issues/1*

* [x] Remove all `_mock`
